Source: written by Calvin Walton <calvin.walton@kepstin.ca>
Upstream: no, hacky workaround for a proprietary upstream
Reason: Remove hardcoded gtk+-1 check that triggers when no gtk+-2 is found
 and frontend is disabled. (This removes all gtk+-1 support)

diff --git a/configure.ac b/configure.ac
index 08354cf..aceaf1b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -107,9 +107,6 @@ PKG_CHECK_MODULES(GTK, gtk+-2.0,
    [
     AC_DEFINE([HAVE_GTK_2], 1,
 	      [Define to 1 if you have the gtk+-2.0 library.])
-   ],[
-    PKG_CHECK_MODULES(GTK, gtk+) 
-    PKG_CHECK_MODULES(GDK_IMLIB, imlibgdk)
    ])
 
 AC_ARG_ENABLE(dependency-reduction,
